bench-tps/src/cli.rs

```diff
@@ -577,6 +587,16 @@ pub fn parse_args(matches: &ArgMatches) -> Result<Config, &'static str> {
         args.client_node_id = Some(client_node_id);
     }
 
+    if let Some(commitment_config) = matches.value_of("commitment_config") {
+        // the * in the match will never be triggered. Validation done by ArgMatches
+        args.commitment_config = match commitment_config {
+            "processed" => CommitmentConfig::processed(),
+            "confirmed" => CommitmentConfig::confirmed(),
+            "finalized" => CommitmentConfig::finalized(),
+            _ => CommitmentConfig::processed(),
+        };
+    }
```

Comment:
`CommitmentConfig` implements `FromStr`; so you can use the convenience macro here to avoid having a copy of these strings here. Moreso, with my suggestion to set `.default_value()`, there will always be a value, even if the user doesn't pass the flag. So, using the `_or_exit!()` variant is fine instead of matching for `Some(...)`
https://docs.rs/clap/2.34.0/clap/macro.value_t_or_exit.html
```suggestion
    let commitment_config = value_t_or_exit!(matches, "commitment_config", CommitmentConfig);
```